src/main/java/org/skife/jdbi/v2/SQLStatement.java

```diff
@@ -245,11 +245,6 @@ protected String getSql()
         return sql;
     }
 
-    protected Binding getParameters()
-    {
-        return params;
-    }
-
```

Comment:
Fair point.  We can block it if you like, it's only mildly silly, just close it if you feel that way.